packages/providers/onekey-ton-provider/src/OnekeyTonProvider.ts

```
@@ -209,34 +241,37 @@
   private _id = 0;
   async _connect(protocolVersion?: number, message?: ConnectRequest): Promise<ConnectEvent> {
     const id = ++this._id;
-    const isGetTonAddr = !message || (message && message.items.some((item) => item.name === "ton_addr"));
-    const proofItem = message && message.items.find((item) => item.name === "ton_proof") as (TonProofItem | undefined);
+    const isGetTonAddr =
+      !message || (message && message.items.some((item) => item.name === 'ton_addr'));
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)